SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -205,6 +205,19 @@
     </PropertyGroup>
   </Target>
 
+  <Target Name="CleanProjectFolders" AfterTargets="Clean" Condition=" $(SonarQubeTempPath) != '' ">
+    <ItemGroup>
+      <ProjectSpecificOutFiles Include="$(SonarQubeOutputPath)\$(MSBuildProjectName)_*\*.*"/>
```

Comment:
A lot of fighting with this. It seems like it's impossible to simply include directories using wildcards